Submitted By: Ronald Hummelink <ronald at hummelink dot xs4all dot nl>
Date: 2003-08-16
Initial Package Version: 1.4.3
Origin: Dejagnu mailinglist: http://mail.gnu.org/archive/html/dejagnu/2003-05/msg00003.html
Description: Fix from H.J. Lu for a bug where the wrong runtest.exp was picked up

2003-05-09  H.J. Lu <hongjiu.lu at intel dot com>

	* runtest: Fix a typo.

--- dejagnu/runtest.break	2002-02-09 22:17:36.000000000 -0800
+++ dejagnu/runtest	2003-05-09 11:54:50.000000000 -0700
@@ -104,8 +104,8 @@ for i in `echo ${execpath} | sed  -e 's@
 	runpath=$i
 	if expr $verbose \> 0 > /dev/null ; then
 	    echo Using $i/runtest.exp as main test driver
-	    break;
 	fi
+	break;
     fi
 done
 # check for an environment variable
